A3A/addons/core/Params.hpp

```diff
@@ -861,6 +861,14 @@ class Params
         texts[] = {$STR_antistasi_dialogs_generic_button_yes_text,$STR_antistasi_dialogs_generic_button_no_text};
         default = 0;
     };
+    class allowUnlockedTNVG: EquipmentParams
+    { 
+        attr[] = {"server"};
+        title = $STR_params_allowUnlockedTNVG;
+        values[] = {1,0};
+        texts[] = {$STR_antistasi_dialogs_generic_button_yes_text,$STR_antistasi_dialogs_generic_button_no_text};
+        default = 1;
```

Comment:
```suggestion
        default = 0;
```
If the param is quite niche, it's usually more harmful making it enabled by default